rotkehlchen/chain/evm/decoding/gearbox/gearbox_cache.py

```diff
@@ -277,8 +293,19 @@ def query_gearbox_data_from_chain(
     pools_data = evm_inquirer.contracts.contract(
         CHAIN_ID_TO_DATA_COMPRESSOR[evm_inquirer.chain_id],
     ).call(node_inquirer=evm_inquirer, method_name='getPoolsV3List')
-    new_pools = []
+    new_pools: list[GearboxPoolData] = []
+    last_notified_ts = Timestamp(0)
+    chain_name = evm_inquirer.chain_id.to_name()
     for pool_data in pools_data:
+        last_notified_ts = maybe_notify_cache_query_status(
+            msg_aggregator=msg_aggregator,
+            last_notified_ts=last_notified_ts,
+            protocol=CPT_GEARBOX,
+            chain_name=chain_name,
+            processed=0,  # 0 because here we only query pools to find the total number of them, it will be incremented while processing the pools in `ensure_gearbox_tokens_existence`  # noqa: E501
```

Comment:
```suggestion
            processed=0,  # 0 because here we only query pools to find their total number. It will be incremented while processing the pools in `ensure_gearbox_tokens_existence`  # noqa: E501
```